### PR TITLE
Notifications Setting QFieldCloud

### PR DESCRIPTION
**Pull request by SeqLaz**
Added Notifications settings in QFieldCloud

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3032 Notifications from QFieldCoud](https://app.clickup.com/t/2192114/QF-3032)

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -230,6 +230,26 @@ Once configured, you can press the cloud button to open the synchronization dial
 
 Now you should see your project and files on [QFieldCloud](https://app.qfield.cloud/)
 
+### Activate Notifications for QFieldCloud changes
+
+1. Access the Settings of your account.
+2. Navigate to the Notifications section. Here, you can customize the frequency of notifications you wish to receive at the valid email address registered with your account.![Synchronize](../../assets/images/frequency_notifications_settings.png)
```
Comment (m-kuhn) [inline documentation/get-started/tutorials/get-started-qfc.en.md:236]: What is the "valid" email address?

**Comment by m-kuhn** [comment]
Thank you

@faebebin can you have a look at this too?

File: documentation/get-started/tutorials/get-started-qfc.en.md (hunk 1 above)
Comment (faebebin) [inline documentation/get-started/tutorials/get-started-qfc.en.md:236]:
```suggestion
2. Navigate to the Notifications section. Here, you can customize the frequency of notifications you wish to receive at the email address registered with your account.![Synchronize](../../assets/images/frequency_notifications_settings.png)
```

Comment (m-kuhn) [reply documentation/get-started/tutorials/get-started-qfc.en.md:236]: Regardless of what exactly we check on code side, I wonder what value the term "valid" in this context will give to the reader of the documentation.

What I understand: a user has registered an email address with his account. This is checked by sending a validation email before a user is activated on the cloud. So in general we _trust_ users that their email address is valid. Referring to "the valid email address" suggests that there's something else like "an invalid email address" which to me sounds confusing without more context.

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 2]
```diff
@@ -230,6 +230,26 @@ Once configured, you can press the cloud button to open the synchronization dial
...
+    - Organization membership created
+    - Organization membership deleted
+    - Team created
+    - Team deleted
+    - Team membership created
+    - Team membership deleted
+    - Project created
+    - Project deleted
+    - Project membership created
+    - Project membership deleted
```
Comment (m-kuhn) [inline documentation/get-started/tutorials/get-started-qfc.en.md:251]: @faebebin how does this list look to you?
Are we able to keep this list updated in the future if more notification events are added?

File: documentation/get-started/tutorials/get-started-qfc.en.md (hunk 1 above)
Comment (faebebin) [reply documentation/get-started/tutorials/get-started-qfc.en.md:236]: Please also include this: 
... does only notify you of events where you are not the actor. You only get notifications for events triggered by your organization members or project collaborators.

@SeqLaz maybe try out collaborators, I know org members work. I did not test if this really works as intended ...


related code snipped:
```
def _concerned_users_in_entity(entity: User):
    """Returns a list of users (of User.Type.PERSON) concerned by updates to an user (any type)"""

    return Person.objects.for_entity(entity)


def _concerned_users_in_project(project: Project):
    """Returns a list of users concerned by updates to a project"""

    return Person.objects.for_project(project).exclude(
        project_role_origin=ProjectQueryset.RoleOrigins.PUBLIC
    )
```

Find it in the repo:
https://github.com/opengisch/qfieldcloud/blob/bbe84ed68372f95ffa73b7f3e186b7d378f439bf/docker-app/qfieldcloud/notifs/signals.py#L24

**Review by faebebin — CHANGES_REQUESTED** [review]

File: documentation/get-started/tutorials/get-started-qfc.en.md (hunk 1 above)
Comment (faebebin) [reply documentation/get-started/tutorials/get-started-qfc.en.md:236]: Yes we should just cross out `valid`

File: documentation/get-started/tutorials/get-started-qfc.en.md (hunk 2 above)
Comment (faebebin) [reply documentation/get-started/tutorials/get-started-qfc.en.md:251]: It is just the list of signal listeneres I copied out of https://github.com/opengisch/qfieldcloud/blob/master/docker-app/qfieldcloud/notifs/signals.py .

I guess we could keep it up to date in future. But I did not actually review / check if they all work in detail.

What is also worth mentioning is that is not configurable at the moment. You get notified for all evetns or none ...

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 3]
```diff
@@ -230,6 +230,26 @@ Once configured, you can press the cloud button to open the synchronization dial
 
 Now you should see your project and files on [QFieldCloud](https://app.qfield.cloud/)
 
+### Activate Notifications for QFieldCloud changes
+
+1. Access the Settings of your account.
+2. Navigate to the Notifications section. Here, you can customize the frequency of notifications you wish to receive at the valid email address registered with your account.![Synchronize](../../assets/images/frequency_notifications_settings.png)
+
+    The notifications event that will create a notification are:
```
Comment (faebebin) [inline documentation/get-started/tutorials/get-started-qfc.en.md:238]:
```suggestion
    The events you get notified about are:
```

File: documentation/get-started/tutorials/get-started-qfc.en.md (hunk 1 above)
Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md]: Done, the functionality in staging works

**Review by SeqLaz — COMMENTED** [review]
Done reviews

**Review by faebebin — APPROVED** [review]
Looks good

Thanks :+1:

**Pull request merged**